### ISSUE TITLE
Create changelog from GitHub release?

### ISSUE THREAD

**Issue by davidbrochart** [agronholm/release-notes#31]
It looks like this action creates a release from a changelog, which requires to populate the changelog manually?
I was wondering if it wouldn't be better to do the opposite: create a GitHub release, which is automatically populated with all the changes since the previous release, and automatically populate a changelog with these changes?

**Comment by agronholm** [comment]
Where would the changes come from?

**Comment by davidbrochart** [comment]
GitHub can automatically [generate release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release):
'In the "Describe this release" field, type a description for your release. If you `@mention` anyone in the description, the published release will include a Contributors section with an avatar list of all the mentioned users. Alternatively, you can automatically generate your release notes by clicking **Generate release notes**.'

**Comment by agronholm** [comment]
Yeah, no thanks. I don't want every single typo fix included in the changelog.

**Comment by agronholm** [comment]
To clarify, the changelogs in my projects are carefully curated to ensure that they only contain changes significant to downstream projects.

**Comment by agronholm** [comment]
The only plausible alternative to this would be [towncrier](https://pypi.org/project/towncrier/), but that too has its own overhead.

**Comment by davidbrochart** [comment]
> To clarify, the changelogs in my projects are carefully curated to ensure that they only contain changes significant to downstream projects.

Makes sense, thanks!
